stock_picking_portal/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Picking Portal",
+    "summary": "Show customer delivery orders in portal",
+    "version": "16.0.1.0.0",
+    "depends": ["stock", "portal", "sale", "website"],
```

Comment:
Why are you adding `website` as dependency?
You don't need to add `sale` and `stock` separately, there is a `sale_stock` module for that.
You don't need to inherit `portal` either because it's [already inherited](https://github.com/odoo/odoo/blob/9141792ee5eed34b4c7f7bda84be92b35df67bad/addons/sale/models/sale_order.py#L37) by `sale`.

Reply:
Thanks, I'll fix it